pkg/valid/id.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 var (
-	startsWithLetter   = regexp.MustCompile(`^[a-z]`)
+	containsLetter     = regexp.MustCompile(`[a-z]`)
```

Comment:
This was a false assumption on my part. There are indeed cluster IDs starting with numbers.